code/modules/cargo/console.dm

```diff
@@ -120,47 +118,28 @@
 	else if (use_beacon && !canBeacon)
 		message = "BEACON ERROR: MUST BE EXPOSED"//beacon's loc/user's loc must be a turf
 	data["message"] = message
-	if(!supply_pack_data)
+	if(outpost_docked)
 		generate_pack_data()
```

Comment:
Would definitely rather this be cached on the console, or at the very least moved to ui_static_data(), since it doesn't look like it ever updates. No need to do all that work every time if the result is going to be the same

Reply:
done, works fine